services/slam/server_test.go

```diff
@@ -302,6 +321,17 @@ func TestFailingServer(t *testing.T) {
 		test.That(t, respInfo, test.ShouldBeNil)
 	})
 
+	t.Run("failing GetProperties", func(t *testing.T) {
+		injectSvc.PropertiesFunc = func(ctx context.Context) (slam.Properties, error) {
+			return slam.Properties{}, errors.New("failure to get properties")
+		}
+		reqInfo := &pb.GetPropertiesRequest{Name: testSlamServiceName}
+
+		respInfo, err := slamServer.GetProperties(context.Background(), reqInfo)
+		test.That(t, err.Error(), test.ShouldContainSubstring, "failure to get properties")
```

Comment:
same comment about making a variable for this error and using test.ShouldBeError